api/models/trial.js

```diff
@@ -210,6 +221,17 @@ const Trial = BaseModel.extend({
 
       return discrepancies;
     },
+    is_registered() {
+      // Be aware that this depends on `source_id` to be accurate
+      // So if a trial has a `nct` source but a `pubmed` source_id,
+      // then the problem is with the processor
+      let status;
+      const source = this.sources[this.attributes.source_id] || { type: null };
+      if (source.type !== null) {
+        status = source.type === 'register';
+      }
+      return status;
```

Comment:
I think it would be better to remove this comment. Here we assume all data is correct, otherwise we're displaying incorrect information.

This code could also be simplified to:

```javascript
const source = this.sources[this.attributes.source_id] || {};
return (source.type === 'register');
```

Without loss of readability IMHO. Also, in general, it's clearer to use parenthesis when writing statements like `a = b == c`, as it's a tricky one (i.e. is it `(a = b) == c` or `a = (b == c)`?)